{{ cookiecutter.project_slug }}/requirements/base.in

```diff
@@ -1,5 +1,5 @@
 # Main requirements of your plugin application.
 
-Django==1.11.20
+Django<2.0
```

Comment:
If the plugin will run in the same Open edX virtualenv, will this break the requirements of the platform? @felipemontoya

Reply:
Since we only use this for testing it wont. But It will stop sending me alerts because one of the django depencencies has a security update

Reply:
Got it.